factory/glideFactoryMonitorAggregator.py

```diff
@@ -265,14 +268,22 @@ def aggregateStatus(in_downtime):
         # load entry status file
         status_fname=os.path.join(os.path.join(monitorAggregatorConfig.monitor_dir, 'entry_'+entry),
                                   monitorAggregatorConfig.status_relname)
+        # load entry completed data file
+        completed_data_fname=os.path.join(os.path.join(monitorAggregatorConfig.monitor_dir,'entry_'+entry),
+                                  monitorAggregatorConfig.completed_data_relname)
         try:
             entry_data=xmlParse.xmlfile2dict(status_fname)
+            completed_data_fp=open(completed_data_fname)
```

Comment:
you probably wanna add a finally clause where you close the file. Or use the with. The risk of file leaking is high otherwise.

Reply:
Done